quickstarts/completions/src/index-key.ts

```diff
@@ -0,0 +1,47 @@
+import { AzureOpenAI } from "openai";
+import { type Completion } from "openai/resources/index";
+import "dotenv/config";
+
+// You will need to set these environment variables or edit the following values
+const endpoint = process.env["AZURE_OPENAI_ENDPOINT"] || "<endpoint>";
+const apiKey = process.env["AZURE_OPENAI_API_KEY"] || "<api key>";
+
+// Azure OpenAI API version and deployment
+const apiVersion = "2024-04-01-preview";
+const deployment = "gpt-35-turbo-instruct";
+
+// Chat prompt and max tokens
+const prompt = ["When was Microsoft founded?"];
+const maxTokens = 128;
+
+function getClient(): AzureOpenAI {
+  return new AzureOpenAI({ endpoint, apiKey, apiVersion, deployment });
+}
+async function getCompletion(
+  client: AzureOpenAI,
+  prompt: string[],
+  model: string,
+  max_tokens: number
+): Promise<Completion> {
+  return await client.completions.create({
+    prompt,
+    model,
+    max_tokens,
+  });
+}
+async function getChoices(completion: Completion): Promise<void> {

```

Comment:
Should it be printChoices?
```suggestion
async function printChoices(completion: Completion): Promise<void> {

```